lib/va_profile/v2/contact_information/service.rb

```diff
@@ -213,9 +213,10 @@ def get_person_transaction_status(transaction_id)
 
         private
 
-        def icn_with_aaid
+        def uuid_with_aaid
           return "#{@user.idme_uuid}^PN^200VIDM^USDVA" if @user.idme_uuid
           return "#{@user.logingov_uuid}^PN^200VLGN^USDVA" if @user.logingov_uuid
+          return "#{vet360_id}^PI^200VETS^USDVA" if @user.idme_uuid.blank? && @user.logingov_uuid.blank?
```

Comment:
Could you use `VA_PROFILE_ID_POSTFIX` from line 16 instead of `^PI^200VETS^USDVA`?

Reply:
I believe this will go away in the refactor. I am ensuring we do not need the vet360_id in the future.